packages/subgraph/tests/allowlist-voting/allowlist-voting.test.ts

```diff
@@ -229,17 +228,17 @@ test('Run Allowlist Voting (handleVoteExecuted) mappings with mock event', () =>
   handleVoteExecuted(event);
 
   // checks
-  assert.fieldEquals('AllowlistProposal', entityID, 'id', entityID);
-  assert.fieldEquals('AllowlistProposal', entityID, 'executed', 'true');
+  assert.fieldEquals('AddresslistProposal', entityID, 'id', entityID);
+  assert.fieldEquals('AddresslistProposal', entityID, 'executed', 'true');
 
   clearStore();
 });
 
-test('Run Allowlist Voting (handleConfigUpdated) mappings with mock event', () => {
+test('Run Addresslist Voting (handleConfigUpdated) mappings with mock event', () => {
   // create state
   let entityID = Address.fromString(VOTING_ADDRESS).toHexString();
-  let erc20VotingPackage = new AllowlistPackage(entityID);
-  erc20VotingPackage.save();
+  let erc20VotingPlugin = new AddresslistPlugin(entityID);
```

Comment:
rename `erc20VotingPlugin` to `addresslistPlugin`

Reply:
Amended